.github/workflows/stale.yml

```diff
@@ -1,209 +1,46 @@
+          stale-pr-label: 'stale'
+          exempt-pr-labels: 'keep' # Don't mark PR's with this label as stale
+          labels-to-remove-when-unstale: 'stale'
           exempt-draft-pr: true
-          exempt-pr-labels: "blocked,must,should,keep,:busts_in_silhouette: discussion,:eyes: Needs Review"
-          delete-branch: false  # if true, job needs permissions "contents: write"
-          # disable issues processing at all (this step is for treat PRs)
-          days-before-issue-stale: -1
-          days-before-issue-close: -1
-          ignore-issue-updates: true
-          remove-issue-stale-when-updated: false
-          stale-issue-label: " "
-
-      - name: Print outputs for PRs
-        run: echo ${{ format('{0},{1}', toJSON(steps.stale-prs.outputs.staled-issues-prs), toJSON(steps.stale-prs.outputs.closed-issues-prs)) }}
-
-      ## Removing private properties from each JSON object and compute array length
-      ## TODO: Delete these set-* workarounds when resolve actions/stale#806 ?
-      - name: Set staled
-        id: set-staled
-        run: |
-          echo $INPUT_ISSUES  \
-            | jq --compact-output --raw-output 'del(.[] | .[to_entries[] | .key | select(startswith("_"))])'  \
-            | sed -e 's/^/issues=/'  \
-            >> $GITHUB_OUTPUT
-          echo $INPUT_ISSUES  \
-            | jq --raw-output '. | length'  \
-            | sed -e 's/^/issues-len=/'  \
-            >> $GITHUB_OUTPUT
-
-          echo $INPUT_PRS  \
-            | jq --compact-output --raw-output 'del(.[] | .[to_entries[] | .key | select(startswith("_"))])'  \
-            | sed -e 's/^/prs=/'  \
-            >> $GITHUB_OUTPUT
-          echo $INPUT_PRS  \
-            | jq --raw-output '. | length'  \
-            | sed -e 's/^/prs-len=/'  \
-            >> $GITHUB_OUTPUT
-        env:
-          INPUT_ISSUES: ${{ steps.stale-issues.outputs.staled-issues-prs }}
-          INPUT_PRS:    ${{ steps.stale-prs.outputs.staled-issues-prs }}
-      - name: Set closed
-        id: set-closed
-        run: |
-          echo $INPUT_ISSUES  \
-            | jq --compact-output --raw-output 'del(.[] | .[to_entries[] | .key | select(startswith("_"))])'  \
-            | sed -e 's/^/issues=/'  \
-            >> $GITHUB_OUTPUT
-          echo $INPUT_ISSUES  \
-            | jq --raw-output '. | length'  \
-            | sed -e 's/^/issues-len=/'  \
-            >> $GITHUB_OUTPUT
-
-          echo $INPUT_PRS  \
-            | jq --compact-output --raw-output 'del(.[] | .[to_entries[] | .key | select(startswith("_"))])'  \
-            | sed -e 's/^/prs=/'  \
-            >> $GITHUB_OUTPUT
-          echo $INPUT_PRS  \
-            | jq --raw-output '. | length'  \
-            | sed -e 's/^/prs-len=/'  \
-            >> $GITHUB_OUTPUT
-        env:
-          INPUT_ISSUES: ${{ steps.stale-issues.outputs.closed-issues-prs }}
-          INPUT_PRS:    ${{ steps.stale-prs.outputs.closed-issues-prs }}
-
-      - name: Write job summary
```

Comment:
The stale action already has a summary build in. We just need to set `ACTIONS_STEP_DEBUG` to true in the repo settings. 

https://github.com/actions/stale#debugging